migration/v0.16.x-v0.17.x/migrate-openid.sh

```diff
@@ -9,6 +9,9 @@ set -euo pipefail
 # Restart the master pods so that they mount the latest version of the secret after apply
 "${here}/../../bin/ck8s" ops kubectl sc delete pod -n elastic-system -l role=master
 
+# Wait for master pod to be ready before updating the config
+sleep 60
```

Comment:
I don't really like using sleep, would it be better to just add this as snippets in the migration doc? And just write "wait until ready" instead of using sleep?

Reply:
We could also have something like this:
```suggestion
kubectl wait --for=condition=ready pod  -n elastic-system opendistro-es-master-0
```

Reply:
The best thing I can come up with atm is a hybrid between both of them. Just the condition alone is not enough as the ports are having some delay in the first seconds.

Reply:
I'm ok with that